plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -181,6 +152,10 @@ export default class NodesTable extends React.Component<
   render() {
     const { data, sortColumn, sortDirection } = this.state;
 
+    if (data.length === 0) {
+      return <Loader />;
```

Comment:
What if there's no nodes? Will it be stuck with the loader?

Reply:
Yes, you are right. I will think of something.

Reply:
Actually, no. If there are no nodes, it will display the no nodes message. 
![screenshot from 2019-01-17 15-57-00](https://user-images.githubusercontent.com/40791275/51323333-8e71e900-1a70-11e9-81b9-59d155ec7040.png)

To test, change this line - https://github.com/dcos/dcos-ui/blob/91c0d1df7d22dbdf450f09c141cde1c17237d7bf/plugins/nodes/src/js/pages/NodesAgents.js#L305 .